doc-changelog/action.yml

```diff
@@ -91,7 +99,60 @@ runs:
       run: |
         python -m pip install --upgrade pip towncrier==${{ inputs.towncrier-version }}
 
+    - name: "Check pull-request title follows conventional commits style"
+      id: lowercase
+      if: ${{ inputs.use-labels == 'false' && ((github.event_name == 'pull_request_target') || (github.event_name == 'pull_request')) }}
+      uses: amannn/action-semantic-pull-request@v5
+      env:
+        GITHUB_TOKEN: ${{ inputs.token }}
+      continue-on-error: true
+
+    - name: "Check pull-request title follows conventional commits style with upper case"
+      id: caps
+      if: ${{ inputs.use-labels == 'false' && ((github.event_name == 'pull_request_target') || (github.event_name == 'pull_request')) }}
+      uses: amannn/action-semantic-pull-request@v5
+      env:
+        GITHUB_TOKEN: ${{ inputs.token }}
+      with:
+        types: |
+          BUILD
+          CHORE
+          CI
+          DOCS
+          FEAT
+          FIX
+          PERF
+          REFACTOR
+          REVERT
+          STYLE
+          TEST
+      continue-on-error: true
```

Comment:
Here, I'm wondering what would be best between running both jobs (like now) or running the most coherent one - e.g. determining if the first letter is lower case or not and run a single job accordingly. This would add complexity to the workflow but diminish resource consumption.

Reply:
Ohh yeah, great point. I think it makes more sense to check the first letter and run the test based on which it is